cmd/influx/user_resource_mapping.go

```diff
@@ -3,10 +3,74 @@ package main
 import (
 	"context"
 	"fmt"
+	"os"
 
 	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/cmd/influx/internal"
 )
 
+func membersListF(filter influxdb.UserResourceMappingFilter, flgs Flags) error {
+	ctx := context.Background()
+	mappingSvc, err := newUserResourceMappingService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize members service client: %v", err)
+	}
+	mps, _, err := mappingSvc.FindUserResourceMappings(ctx, filter)
+	if err != nil {
+		return fmt.Errorf("failed to find members: %v", err)
+	}
+	userSVC, err := newUserService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize users service client: %v", err)
+	}
+	urs := make([]*influxdb.User, len(mps))
+	ursC := make(chan struct {
+		User  *influxdb.User
+		Index int
+	}, len(mps))
+	errC := make(chan error)
+	for k, v := range mps {
+		go func(k int, v *influxdb.UserResourceMapping) {
+			usr, err := userSVC.FindUserByID(ctx, v.UserID)
```

Comment:
just out of curiosity, is it possible to provide a `List` operation for the user service? then let that manage the concurrency? I'm not sure if we can obtain a list in one call or not, just curious 🤷‍♂

Reply:
Yeah, I initially thought about that as well. But that requires API schema change.